### PR TITLE
Fix typos and minor inconsistencies

### PR DESCRIPTION
**Pull request by jhrcek**

### CODE REVIEW

File: src/Lazy/LList.elm [hunk 1]
```diff
@@ -236,11 +216,11 @@ This function is performed lazily.
 
 -}
 map : (a -> b) -> LList a -> LList b
-map predicate =
-    Lazy.map (List.map predicate)
+map f =
```
Comment (jhrcek) [inline src/Lazy/LList.elm]: predicate is commonly used to mean `a -> Bool`

File: package.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
     "test": "./node_modules/.bin/elm-verify-examples --elm-test=elm-test"
   },
   "dependencies": {
-    "elm": "^0.19.0-bugfix2",
-    "elm-test": "^0.19.0-beta11",
-    "elm-verify-examples": "^3.0.1"
+    "elm": "^0.19.0-bugfix6",
```
Comment (jhrcek) [inline package.json]: Bump to latest versions (this has the cascading effects in the package-lock.json)

File: src/Lazy/LList.elm (hunk 1 above)
Comment (turboMaCk) [reply src/Lazy/LList.elm]: agree

File: package-lock.json [hunk 3]
```diff
@@ -3249,26 +3568,26 @@
...
     "yargs-parser": {
       "version": "8.1.0",
       "resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-8.1.0.tgz",
       "integrity": "sha512-yP+6QqN8BmrgW2ggLtTbdrOyBNSI7zBa4IykmiV5R1wl1JWNxQvWhMfMdmzIYtKU7oP3OOInY/tl2ov3BDjnJQ==",
       "requires": {
-        "camelcase": "^4.1.0"
+        "camelcase": "4.1.0"
       }
     }
   }
```
Comment (turboMaCk) [inline package-lock.json]: npm style 🙄

**Pull request merged**